lib/thrift/generator/client.ex

```diff
@@ -0,0 +1,274 @@
+defmodule Thrift.Generator.Client do
+  defmodule Framed do
+    alias Thrift.Generator.Service
+    alias Thrift.Parser.Models.Function
+
+    def generate(service_module, service) do
+      functions = service.functions
+      |> Map.values
+      |> Enum.map(&generate_function(service_module, &1))
+
+      quote do
```

Comment:
That's true, but I think that having the additional functions adds a lot of value. 
Perhaps I have a wrapper module that just starts the connection process and sends messages to it?